### ISSUE TITLE
AboutModal image alt text

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-react#1008]
For the image that displays on the right side of the About Modal, the following alt text is defined:
"AboutModal Logo"

Instead, we should update this to represent best practice (i.e. describe the image as a "logo", but also repeat any text included in the image). The following text should be used instead: 
"Patternfly Logo"
